mne/preprocessing/maxwell.py

```diff
@@ -502,6 +579,7 @@ def _sss_basis(origin, coils, int_order, ext_order, mag_scale=100.,
                          for coil in coils])] = mag_scale
 
     # Compute internal/external basis vectors (exclude degree 0; L/RHS Eq. 5)
+    mu_0 = 4e-7 * np.pi  # magnetic permeability
```

Comment:
raw data:
![raw_data](https://cloud.githubusercontent.com/assets/2365790/10978247/ceffacd8-83c3-11e5-8dce-d26c73e6b1ae.png)

with `maxwell_filter(raw, cross_talk=ctc_fname, calibration=fine_cal_fname)`:
![maxwell_filtered_data](https://cloud.githubusercontent.com/assets/2365790/10978257/dc0c4e54-83c3-11e5-967a-47fde6bb5de4.png)


with an equivalent call to `maxfilter` as `maxfilter -f sample_audvis_raw.fif -o sample_audvis_raw_sss.fif -ctc ct_sparse_mgh.fif -cal sss_cal_mgh.dat -regularize in -autobad off -iterate 0 -bad 2443 1032 2313` (iterate and regularize are default values for my setup, not sure about MGH):
![maxfilter_equivalent_data](https://cloud.githubusercontent.com/assets/2365790/10978367/9317463a-83c4-11e5-9257-6d0e73addf78.png)